Dockerfile.win10.min

```diff
@@ -63,7 +63,6 @@ WORKDIR /
 # option --depth=1 should also be restored
 RUN git clone --single-branch -b master https://github.com/microsoft/vcpkg.git
```

Comment:
Maybe cloning the release? Main is likely to be unstable

Reply:
@GuanLuo - Sure I will close this request and will add a new one for the same.